app/views/users/edit.html.erb

```diff
@@ -1,2 +1,24 @@
-<h1>Users#edit</h1>
-<p>Find me in app/views/users/edit.html.erb</p>
+<div class="containter">
+	<div class="row">
+		<div class="col-sm-6 col-sm-offset-3 user_signup_form">
+			<%= form_for @user, url: users_path, html: { multipart: true } do |f| %>
```

Comment:
You're really close, but you'll want to do `url: user_path(@user)` to build the correct route `/users/:id`.

Also, since `@user` is an instance of `Student`, `<%= form_for @user` is building the form params like this: `student[name]`, `student[email]`, etc. You want these to be `user` params instead, so change the form to `<%= form_for :user`.

After making that change, you'll have to set `method: :patch` in the form as well. So the whole thing looks like this:

```html
<%= form_for :user, url: user_path(@user), method: :patch, html: { multipart: true } do |f| %>
```

We can definitely talk more about this tomorrow! :smile: